namespaces/kubernetes.yml

```diff
@@ -6,55 +6,49 @@ namespace:
     Namespace for kubernetes-specific metadata
   fields:
   - name: pod_name
-    type: string
-    norms:
-      enabled: true
+    type: keyword
+    norms: true
     description: >
       The name of the pod
 
   - name: pod_id
-    type: string
-    norms:
-      enabled: true
+    type: keyword
+    norms: true
     description: >
       Kubernetes ID of the pod.
 
   - name: namespace_name
-    type: string
-    norms:
-      enabled: true
+    type: keyword
+    norms: true
     description: >
       The name of the namespace in Kubernetes.
 
   - name: namespace_id
-    type: string
-    norms:
-      enabled: true
+    type: keyword
+    norms: true
     description: >
       ID of the namespace in Kubernetes.
 
   - name: host
-    type: string
-    norms:
-      enabled: true
+    type: keyword
+    norms: true
     description: >
       Kubernetes node name
 
   - name: master_url
-    type: string
+    type: keyword
     description: >
       Kubernetes Master URL
 
   - name: container_name
-    type: string
-    norms:
-      enabled: true
+    type: text
+    norms: true
     description: >
       The name of the container in Kubernetes.
     fields:
       - name: raw
         ignore_above: 256
-        type: string
+        type: keyword
 
   - name: annotations
```

Comment:
not sure that this is a right forum to discuss this, but now that ES has limit of 1000 fields per index by default we will easily hit the limit the way we store annotations and labels and then logging will stop.

/cc @portante